docs/install.md

```diff
@@ -22,6 +22,16 @@ To be able to clone the repo and run the application you need:
 
 ## Installing and Running
 
+### Ubuntu and [snap supported distros](https://snapcraft.io/docs/core/install) 
```

Comment:
Having the section last makes sense - and I'd like to see it under a new heading to point out it's a pre-packaged/compiled offering. Also please point out in the instructions that we're not officially supporting the distribution channel (until we decide to publish the releases ourselves). So something like:

---

### Pre-Packaged Distributions

#### Official Direct Downloads

Install the officially released desktop app by visiting [https://apps.wordpress.com/desktop/](https://apps.wordpress.com/desktop/). Download the archive for your platform of choice and install manually.

#### Unofficial Linux (Ubuntu & others) via [snap](https://snapcraft.io/docs/core/install)
...

---

Please update your PR to create this new section, replace the heading for snap, and put this as the last part of the readme. Thanks!